students/bcoates/lesson03/mailroom.py

```diff
@@ -79,10 +76,10 @@ def prompt_donation_amount(full_name):
             float_amount = float(donation_amount)
 
             # Add the donation to the donor's list of donations
-            for donors in donor_history:
-                if donors[0] == full_name:
+            for donor in donor_history:
+                if donor[0] == full_name:
                     print("Adding new donation of {} from {}".format(donation_amount, full_name))
-                    donors[1].append(float_amount)
+                    donor[1].append(float_amount)
```

Comment:
I still think you can improve your program by working with donor item from the list instead of the name since you're looping through your structure all over again when have already found the donor previously (see last PR comments).